### DISCUSSION TITLE
MQTT client library?

### DISCUSSION THREAD

**Discussion by infn-ke** [nanomq/nanomq#262] [Q&A] [answered]
Is there any MQTT client library included with nanoMQ?

**Comment by JaylinYu** [comment, marked as answer]
Please check branch here:
https://github.com/nanomq/NanoSDK
you can have a try on this demo:
https://github.com/nanomq/NanoSDK/tree/main/demo/mqtt_async

**Reply by JaylinYu** [reply]
Docs:
https://github.com/nanomq/NanoSDK/tree/main/docs
